### ISSUE TITLE
Theme views are overridden by plugin views when in another plugin

### ISSUE THREAD

**Issue by zerocrates** [omeka/Omeka#635]
Normally, the way views are loaded looks like this:

* current theme
* Omeka defaults
* plugins

When in another module (so, a page added by a plugin), the order shifts to:

* theme overrides for current plugin
* current plugin
* other plugins
* current theme
* Omeka defaults

When you're in a view added by a plugin, you can still override *those* views, by putting them in the plugin-specific folder of your theme (like `my-theme/exhibit-builder`). But, overrides to things in places like the common/ directory which work correctly on normal pages don't on any plugin page, since the theme views are shifted to the bottom of the stack. So, any common view provided by a plugin takes precedence over whatever's in the theme.

To fix this, we need to change the "in a plugin" reordering to look like this:

* theme overrides for current plugin
* current plugin
* current theme
* Omeka defaults
* other plugins

Basically, we want to make it so that *only* the current plugin gets special treatment (since there's a specific mechanism for theme writers to override in that case), and the other plugins stay where they normally are, at the bottom of the stack.
